Update tasklist.c to fix incompatible pointer type error:

| libwnck/tasklist.c:4355:49: error: passing argument 2 of 'sn_startup_sequence_get_last_active_time'
     from incompatible pointer type [-Wincompatible-pointer-types]
|  4355 |                                                 &tv_sec, &tv_usec);
|       |                                                 ^~~~~~~
|       |                                                 |
|       |                                                 long int *

Upstream-Status: Inactive-Upstream

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 libwnck/tasklist.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libwnck/tasklist.c b/libwnck/tasklist.c
index 4f0c6c4..d3bdf96 100644
--- a/libwnck/tasklist.c
+++ b/libwnck/tasklist.c
@@ -4352,7 +4352,7 @@ sequence_timeout_callback (void *user_data)
       WnckTask *task = WNCK_TASK (tmp->data);
 
       sn_startup_sequence_get_last_active_time (task->startup_sequence,
-                                                &tv_sec, &tv_usec);
+                                                (time_t *)&tv_sec, &tv_usec);
       
       elapsed =
         ((((double)now.tv_sec - tv_sec) * G_USEC_PER_SEC +
